<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIC2TEXT</class>
 <widget class="QMainWindow" name="PIC2TEXT">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1471</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIC2TEXT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ICONS/LOGO.png</normaloff>../ICONS/LOGO.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>9.000000000000000</double>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="downloadB">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>590</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;click to convert &lt;/p&gt;&lt;p&gt;file to text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ICONS/DOWNLOAD.png</normaloff>../ICONS/DOWNLOAD.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>201</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>1341</width>
      <height>561</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:48pt; font-weight:600; font-style:italic; color:#4d051c; vertical-align:middle;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#4d051c; vertical-align:middle;&quot;&gt;L&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#9a0a38; vertical-align:middle;&quot;&gt;o&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#326a32; vertical-align:middle;&quot;&gt;a&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#810652; vertical-align:middle;&quot;&gt;d&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#146a28; vertical-align:middle;&quot;&gt;i&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#a5874b; vertical-align:middle;&quot;&gt;n&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#64b396; vertical-align:middle;&quot;&gt;g&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#656a33; vertical-align:middle;&quot;&gt;.&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#6a0726; vertical-align:middle;&quot;&gt;.&lt;/span&gt;&lt;span style=&quot; font-size:48pt; font-weight:600; font-style:italic; color:#14184d; vertical-align:middle;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
    <property name="tabStopDistance">
     <double>80.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Click here to go back.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../ICONS/arrow-back-icon.png</normaloff>../ICONS/arrow-back-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
